editor/plugins/texture_editor_plugin.cpp

```diff
@@ -70,7 +70,20 @@ void TexturePreview::_update_metadata_label_text() {
 		format = texture->get_class();
 	}
 
-	metadata_label->set_text(vformat(String::utf8("%s×%s %s"), itos(texture->get_width()), itos(texture->get_height()), format));
+	const Ref<Image> image = texture->get_image();
+	const int mipmaps = image->get_mipmap_count();
+	int memory = image->get_width() * image->get_height() * Image::get_format_pixel_size(image->get_format());
```

Comment:
I'm concerned that there may be slightly more to calculating the memory used then this. It looks like ``get_format_pixel_size`` returns ``1`` for all compressed formats, which I find hard to believe. Maybe @lyuma  knows more.